### DISCUSSION TITLE
Hash+Salt tokens in db strategy

### DISCUSSION THREAD

**Discussion by lemanjo** [fastapi-users/fastapi-users#1322] [General]
I'm currently trying to figure out whether there is already some built-in support for this or not. Cannot seem to find anything from the docs, but if anyone could give some pointers regarding this?

I've implemented the bearer+database auth backend to allow system integrations. In general, I think there should be a way to protect the tokens in db when using a database strategy?

**Comment by frankie567** [comment]
Hi @lemanjo 👋 

That's indeed something that could be done, even though not as critical as password hashing since the value likely won't be reusable in other services.

PR welcome if someone wants to tackle this!
